src/utils/v2v3/loadV2V3Contract.ts

```diff
@@ -41,13 +44,19 @@ export const loadV2V3Contract = async (
   } else if (contractName === V2V3ContractName.JBVeTokenUriResolver) {
     contractJson = await loadVeTokenUriResolver()
   } else {
-    contractJson = await loadJuiceboxV2Contract(contractName, network)
+    contractJson =
+      version === CV_V2
+        ? await loadJuiceboxV2Contract(contractName, network)
+        : version === CV_V3
+        ? await loadJuiceboxV3Contract(contractName, network)
+        : undefined
   }
 
   if (!contractJson) {
-    throw new Error(
-      `Error importing contract ${contractName}. Network: ${network})`,
+    console.error(
```

Comment:
Idk, do we need to throw here? i don' think so.

Reply:
Yeah surely unnecessary.